docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,78 @@
+let signUpPassword = '';
+const verifiedUsers = new Map();
+const FIXED_USERNAME = 'test';
+const FIXED_PASSWORD = 'password';
```

Comment:
Constants should be place above other variable declarations

Reply:
Done!